Mathlib/Analysis/LocallyConvex/AbsConvexOpen.lean

```diff
@@ -0,0 +1,121 @@
+/-
+Copyright (c) 2022 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+import Mathlib.Analysis.LocallyConvex.AbsConvex
+import Mathlib.Analysis.LocallyConvex.WithSeminorms
+import Mathlib.Analysis.Convex.Gauge
+
+/-!
+# Absolutely convex open sets
+
+A set `s` in an commutative monoid `E` is called absolutely convex or disked if it is convex and
+balanced. The importance of absolutely convex sets comes from the fact that every locally convex
+topological vector space has a basis consisting of absolutely convex sets.
```

Comment:
Why is this module doc the same as the other file? They should be distinguished somewhat.